Formula/gstreamer.rb

```diff
@@ -149,6 +150,7 @@ def install
       -Dgst-editing-services:pygi-overrides-dir=#{site_packages}/gi/overrides
       -Dgst-python:pygi-overrides-dir=#{site_packages}/gi/overrides
       -Dgst-python:python=#{python3}
+      -Dgst-plugins-bad:opencv=disabled
```

Comment:
[`openexr` this time](https://github.com/Homebrew/homebrew-core/actions/runs/5107907041/jobs/9181323332?pr=131668#step:4:3973).

```suggestion
      -Dgst-plugins-bad:opencv=disabled
      -Dgst-plugins-bad:openexr=disabled
```

Reply:
:smiling_face_with_tear:

Reply:
Just add it as a dependency, `openexr` is already in the dep tree.

Reply:
Ah, sure.